front/src/modules/simulationResult/components/TrainDetails.tsx

```diff
@@ -30,33 +31,48 @@ const getOccupancyBounds = (
 };
 
 export default function TrainDetails() {
-  const { positionValues, timePosition, consolidatedSimulation, selectedTrainId } = useSelector(
+  const { consolidatedSimulation, selectedTrainId } = useSelector(
     (state: RootState) => state.osrdsimulation
   );
 
+  const [localTimePosition, setLocalTimePosition] = useState<Date>(new Date());
+  const [localPositionValues, setLocalPositionValues] = useState<PositionsSpeedTimes<Date>>(
+    {} as PositionsSpeedTimes<Date>
+  );
+  useChartSynchronizer(
+    (timePosition, positionValues) => {
+      setLocalTimePosition(timePosition);
+      setLocalPositionValues(positionValues);
+    },
+    'train-detail',
+    []
+  );
+
   const { t } = useTranslation(['simulation']);
 
   const occupancyBounds = useMemo(() => {
     const foundTrain = consolidatedSimulation.find((train) => train.id === selectedTrainId);
 
     return (
       foundTrain &&
-      getOccupancyBounds(foundTrain.routeAspects, new Date(positionValues?.headPosition?.time))
+      getOccupancyBounds(foundTrain.routeAspects, new Date(localPositionValues?.headPosition?.time))
     );
-  }, [consolidatedSimulation, positionValues, selectedTrainId]);
+  }, [consolidatedSimulation, localPositionValues, selectedTrainId]);
 
   return (
     <div className="d-flex">
-      {positionValues.headPosition && timePosition && (
+      {localPositionValues?.headPosition && localTimePosition && (
         <>
           <div className="rounded px-1 train-detail small bg-blue text-white text-nowrap mr-2">
             <div className="font-weight-bold text-uppercase">{t('trainDetails.headPosition')}</div>
-            {positionValues.headPosition && Math.round(positionValues.headPosition.position) / 1000}
+            {localPositionValues?.headPosition &&
+              Math.round(localPositionValues.headPosition.position) / 1000}
```

Comment:
Move the condition in each div (same for tailPosition and speed)
```suggestion
      {headPosition && (
        <div className="rounded px-1 train-detail small bg-blue text-white text-nowrap mr-2">
          <div className="font-weight-bold text-uppercase">{t('trainDetails.headPosition')}</div>
          {Math.round(headPosition.position) / 1000}
          km
        </div>
      )}
```

Reply:
new PR for that